# Returns size of cartesian pose buffer
---
int64 size
bool success
string msg
